js/share.js

```diff
@@ -535,12 +444,12 @@
             }
             return this._shareOptionsTemplate(data);
         }
-
     };
 
+    // v10.3
     OCA.Onlyoffice.ShareDialogView = {
         attach: function (view) {
-            if (_.isUndefined(view) || _.isUndefined(view.model) || !!view.model.getRegisteredShareAttribute) {
+            if (_.isUndefined(view) || _.isUndefined(view.model) || OCA.Onlyoffice.setting.shareAttributesVersion !== "v2") {
```

Comment:
I would like to be based on the availability of functionality, but not on the version number of ownCloud. This would not forbid me to release my code. And it is not known which version the new API will get into.